src/box/lua/config/init.lua

```diff
@@ -4,6 +4,7 @@ local cluster_config = require('internal.config.cluster_config')
 local configdata = require('internal.config.configdata')
 local aboard = require('internal.config.utils.aboard')
 local tarantool = require('tarantool')
+local textutils = require('internal.config.utils.textutil')
```

Comment:
Nit: Why `testutil` here and `textutils` there? If one name is better than other, let's choose one that is better. If they both are OK, let's choose any.

Reply:
I prefer `textutils`. Fixed.

Reply:
OK. Fix the commit subject then.

Reply:
Commit message fixed.